CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Change log
 
+## 5.0.0 (2018-06-18)
```

Comment:
Can we merge the changes listed below under an Unreleased heading? That way we can more easily keep up with new pull requests and the imminent 4.8.2 release.

Reply:
Wait, I'm not fully following 😆 
Why would that make it easier? Also, with Reek 5 clooooooose to a release, is this comment still relevant?

Reply:
No, that comment was from a couple of days ago, it is no longer relevant.